packages/gatsby-plugin-preact/src/gatsby-node.js

```diff
@@ -2,13 +2,12 @@ exports.onCreateWebpackConfig = ({ stage, actions }) => {
   // Requiring the server version of React-dom is hardcoded right now
   // in the development server. So we'll just avoid loading Preact there
   // for now.
-  if (stage !== `develop-html`) {
+  if (stage !== `develop`) {
```

Comment:
This shouldn't be necessary as preact is 100% compatible with react 16. There are some issues with HMR & devtools.

Reply:
Didn't dig too far into it, but Preact definitely breaks in dev with Gatsby. Assumed it was to do with HMR et al